sql/core/src/test/scala/org/apache/spark/sql/collation/DefaultCollationTestSuite.scala

```diff
@@ -410,6 +410,70 @@ class DefaultCollationTestSuiteV1 extends DefaultCollationTestSuite {
         // scalastyle:on
       }
     }
+    withView(testView) {
+      // scalastyle:off
+      sql(
+        s"""CREATE OR REPLACE VIEW $testView
+           | (c1)
+           | DEFAULT COLLATION sr_ai
+           | AS SELECT 'Ć' as c1 WHERE 'Ć' = 'C'
+           |""".stripMargin)
```

Comment:
Should 2 spaces indentation, please, tune your IDE. Also see https://github.com/databricks/scala-style-guide?tab=readme-ov-file#indent